contrib/opencensus-correlation/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.0'
```

Comment:
@reyang do you want to keep releasing this package?

Reply:
Up to you (depending on which way is easier for you). We don't have dependency from opencensus and there is no actual functionality, no need to release it.

Once we heard back from @bogdandrutu, we can decide whether to put the correlation context work in this package, or remove it (and do the implementation in opencensus core lib).

Reply:
OK, I won't touch it in this branch and it'll be released again as the same version.